qa-tests-backend/src/main/groovy/orchestratormanager/Kubernetes.groovy

```diff
@@ -297,11 +299,11 @@ class Kubernetes implements OrchestratorMain {
     void deleteAllPodsAndWait(String ns, Map<String, String> labels) {
         log.debug "Will delete all pods in ${ns} with labels ${labels} and wait for deletion"
 
-        List<Pod> beforePods = evaluateWithRetry(2, 3) {
+        List<Pod> beforePods = Helpers.evaluateWithRetry(null, 2, 3) {
             client.pods().inNamespace(ns).withLabels(labels).list().getItems()
         }
         beforePods.each { pod ->
-            evaluateWithRetry(2, 3) {
+            Helpers.evaluateWithRetry(null, 2, 3) {
```

Comment:
I think we no longer need this `null`. After migration to groovy 3.0 we should be able to remove `Object ignored` from `Helpers`.

Reply:
It did not work without the `null`.
Do you mean we're at 3.0 already and I should extend this change to remove `Object ignored`? Or is this a comment about what will be possible once we migrate to 3.0?

Reply:
OK, seems to compile after removal, so I guess we're on 3.0 already. Let's see if it will work too.

Reply:
Well, it does not actually work @janisz 

Here it fails at runtime with:
```
    groovy.lang.MissingMethodException: No signature of method: static BaseSpecification.withRetry() is applicable for argument types: (Integer, Integer, BaseSpecification$_globalSetup_closure3) values: [10, 1, BaseSpecification$_globalSetup_closure3@236861da]
    	at BaseSpecification.globalSetup(BaseSpecification.groovy:101) [3 skipped]
```

I attempted to fix this occurrence with a static import [here](https://github.com/stackrox/stackrox/pull/5082), but this just uncovered almost a hundred other failures in various tests.

Any ideas? Would it be worth trying to introduce the static import in all those other places? Or should I just bring the `Object ignored` back?